ts/session/apis/open_group_api/sogsv3/sogsV3SendReaction.ts

```diff
@@ -0,0 +1,91 @@
+import { AbortSignal } from 'abort-controller';
+import { Data } from '../../../../data/data';
+import { OpenGroupReactionResponse, Reaction } from '../../../../types/Reaction';
+import { getEmojiDataFromNative } from '../../../../util/emoji';
+import { OnionSending } from '../../../onions/onionSend';
+import { OpenGroupPollingUtils } from '../opengroupV2/OpenGroupPollingUtils';
+import { batchGlobalIsSuccess, parseBatchGlobalStatusCode } from './sogsV3BatchPoll';
+
+export const hasReactionSupport = async (id: number): Promise<boolean> => {
+  const found = await Data.getMessageByServerId(id);
+  if (!found) {
+    window.log.warn(`Open Group Message ${id} not found in db`);
+    return false;
+  }
+
+  const conversationModel = found?.getConversation();
+  if (!conversationModel) {
+    window.log.warn(`Conversation for ${id} not found in db`);
+    return false;
+  }
+
+  if (!conversationModel.hasReactions) {
+    window.log.warn("This open group doesn't have reaction support. Server Message Id", id);
+    return false;
+  }
```

Comment:
you are leaving bugs here!
hasReactions is always true (it is a function which needs to be called!)

Reply:
A thousand apologies sir! 🙏